rules/linux/defense_evasion_env_binary.toml

```diff
@@ -1,11 +1,11 @@
 [metadata]
 creation_date = "2022/02/24"
 maturity = "production"
-updated_date = "2022/02/24"
+updated_date = "2022/03/04"
 
 [rule]
 author = ["Elastic"]
-description = "Identifies Linux binary env abuse to break out from restricted environments by spawning an interactive system shell"
+description = "Identifies Linux binary env abuse to break out from restricted environments by spawning an interactive system shell.This activity is not standard use with this binary for a user or system administrator. It indicates a potentially malicious actor attempting to improve the capabilities or stability of their access"
```

Comment:
can you lint the toml file `python -m detection-rules toml-lint -f \rule\path_to_rule.toml` ?

Reply:
![image](https://user-images.githubusercontent.com/91139415/157048459-63d706d8-3fde-4551-84c0-ff3c1b8037ab.png)
Done